alpha/lib/model/LiveEntry.php

```diff
@@ -313,21 +313,23 @@ public function getLiveStreamConfigurations($protocol = 'http', $tag = null, $cu
 		$primaryApplicationName = null;
 		$backupApplicationName = null;
 		$isExternalMediaServerStream = false;
-		
-		$kMediaServers = $this->getMediaServers();
-		if(count($kMediaServers))
+		/* @var $liveEntryServerNode LiveEntryServerNode */
+		$liveEntryServerNode = null;
+		/* @var $serverNode WowzaMediaServerNode */
+		$serverNode = null;
+
+		$liveEntryServerNodes = $this->getLiveEntryServerNodes();
```

Comment:
This will return live entries which are in status "BROADCASTING" and "AUTHENTICATED" as well ... this changes the previous logic that would have returned kMediaservers only when the status was PLAYABLE.

filter in "getLiveEntryServerNodes" only entryServerNodes with status PLAYABLE